apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationStatusForm/AssetSkillsFiltered.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { Experience, Skill } from "@gc-digital-talent/graphql";
+import Well from "@gc-digital-talent/ui/src/components/Well";
+
+import SkillTree from "~/pages/Applications/ApplicationSkillsPage/components/SkillTree";
+import { getExperiencesSkillIds } from "~/utils/skillUtils";
+
+export interface AssetSkillsFilteredProps {
+  poolAssetSkills: Skill[];
+  experiences: Experience[];
+}
+
+const AssetSkillsFiltered = ({
+  poolAssetSkills,
+  experiences,
+}: AssetSkillsFilteredProps) => {
+  const intl = useIntl();
+
+  // conditional rendering function, dependent on input array length
+  const renderAssetsSkillTree = (assetSkills: Skill[]) => {
+    if (assetSkills.length > 0) {
+      return (
+        <>
+          <p>
+            {intl.formatMessage({
+              defaultMessage: "Represented by the following experiences:",
+              id: "mDowK/",
+              description:
+                "Lead in text for experiences that represent the users skills",
+            })}
+          </p>
+          {assetSkills.map((optionalTechnicalSkill) => (
+            <SkillTree
+              key={optionalTechnicalSkill.id}
+              skill={optionalTechnicalSkill}
+              experiences={experiences}
+              showDisclaimer
+              hideConnectButton
+              hideEdit
+              disclaimerMessage={
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "There are no experiences attached to this skill.",
+                    id: "XrfkBm",
+                    description:
+                      "Message displayed when no experiences have been attached to a skill",
+                  })}
+                </p>
+              }
+            />
+          ))}
+        </>
+      );
+    }
+
+    return (
+      <Well>
+        {intl.formatMessage({
+          defaultMessage: "There are no asset skills attached.",
+          id: "VeiXJP",
+          description:
+            "Informing that a section is empty of any associated asset skills.",
+        })}
+      </Well>
+    );
+  };
+
+  const experiencesSkillIds = getExperiencesSkillIds(experiences);
+  const usedAssetsSkills = poolAssetSkills.filter((assetSkill) =>
+    experiencesSkillIds.includes(assetSkill.id),
+  );
+
+  return <>{renderAssetsSkillTree(usedAssetsSkills)}</>;
```

Comment:
Could this be done with a simple ternary instead of a function?

```tsx
return usedAssetSkillls.length > 0 ? <Tree /> : <NullState />;
```

Reply:
I can try, I was originally doing something like that but the error/linting squigglies were really confusing me 😅

Reply:
Done I believe in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7027/commits/4189331f48dae72e3733ccc08b127a8693b71be4